pyprep/prep_pipeline.py

```diff
@@ -54,16 +54,38 @@ class PrepPipeline:
     def __init__(self, raw, prep_params, montage, ransac=True, random_state=None):
         """Initialize PREP class."""
         self.raw = raw.copy()
-        self.ch_names = self.raw.ch_names
+
+        # separe eeg and non eeg channels
+        self.raw_non_eeg = self.raw.copy()
+        self.chs_all = self.raw.ch_names.copy()
+        self.chs_types = self.raw.get_channel_types()
+        self.chs_eeg = [
+            self.chs_all[i]
+            for i in range(len(self.chs_all))
+            if self.chs_types[i] == "eeg"
+        ]
+        self.chs_non_eeg = set(self.chs_all) - set(self.chs_eeg)
+        self.raw.pick_channels(self.chs_eeg)
```

Comment:
this makes me feel like `self.raw` should actually be `self.raw_eeg`

Reply:
I was wondering the same thing. We should have a discussion on the naming of input output parameters. I remember one user confused becase the "raw" was actually the processed signal. Obviously from the developer pov we are referring to the raw mne object but the user is thinking of an unprocessed signal.

So, should we stick to the raw  prefix?

Now for the suffix: should the full_raw actually be raw_all?

Reply:
Will leave it as 'raw' until we decide the naming convention